java/core/src/test/org/apache/orc/impl/TestRecordReaderImpl.java

```diff
@@ -310,6 +310,9 @@ public Object getLiteral() {
       return null;
     }
 
+    @Override
```

Comment:
Looks like we will do another release of StorageAPI. I was wondering if we should push this as a default?

Reply:
Yes, we need 2.8.1. This PR will stay here and will retarget to 2.8.1, @pavibhai .

BTW, what do you mean by the following?
> we should push this as a default?

Reply:
I was wondering if we should add a default implementation to this new method into PredicateLeaf. But not required if this is the only failure we are seeing.